app.js

```diff
@@ -0,0 +1,188 @@
+const { MongoClient } = require("mongodb");
+
+async function run() {
+  // TODO:
+  // Replace the placeholder connection string below with your
+  // Altas cluster specifics. Be sure it includes
+  // a valid username and password! Note that in a production environment,
+  // you do not want to store your password in plain-text here.
+  const uri =
+    "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+  // The MongoClient is the object that references the connection to our
+  // datastore (Atlas, for example)
+  const client = new MongoClient(uri);
+
+  // The connect() method does not attempt a connection; instead it instructs
+  // the driver to connect using the settings provided when a connection
+  // is required.
+  await client.connect();
+
+  // Provide the name of the database and collection you want to use.
+  // If the database and/or collection do not exist, the driver and Atlas
+  // will create them automatically when you first write data.
+  const dbName = "myDatabase";
+  const collectionName = "recipes";
+
+  // Create references to the database and collection in order to run
+  // operations on them.
+  const database = client.db(dbName);
+  const collection = database.collection(collectionName);
+
+  /*
+   *  *** INSERT DOCUMENTS ***
+   *
+   * You can insert individual documents using collection.insert().
+   * In this example, we're going to create four documents and then
+   * insert them all in one call with collection.insertMany().
+   */
+
+  const recipes = [
+    {
+      name: "elotes",
+      ingredients: [
+        "corn",
+        "mayonnaise",
+        "cotija cheese",
+        "sour cream",
+        "lime",
+      ],
+      prepTimeInMinutes: 35,
+    },
+    {
+      name: "loco moco",
+      ingredients: [
+        "ground beef",
+        "butter",
+        "onion",
+        "egg",
+        "bread bun",
+        "mushrooms",
+      ],
+      prepTimeInMinutes: 54,
+    },
+    {
+      name: "patatas bravas",
+      ingredients: [
+        "potato",
+        "tomato",
+        "olive oil",
+        "onion",
+        "garlic",
+        "paprika",
+      ],
+      prepTimeInMinutes: 80,
+    },
+    {
+      name: "fried rice",
+      ingredients: [
+        "rice",
+        "soy sauce",
+        "egg",
+        "onion",
+        "pea",
+        "carrot",
+        "sesame oil",
+      ],
+      prepTimeInMinutes: 40,
+    },
+  ];
+
+  collection
+    .insertMany(recipes)
+    .then(result =>
+      console.log(`${result.insertedCount} documents successfully inserted.`),
```

Comment:
Can we add a new line before or after each console.log so that there is an empty line between the output  of each command?

Reply:
Sure, so just to confirm, the output you're hoping to see is:

```
4 documents successfully inserted.

elotes has 5 ingredients and takes 35 minutes to make
fried rice has 7 ingredients and takes 40 minutes to make

Here is the updated document: {"_id":"5efd0cf59d927c69d1bbb1a0","name":"patatas bravas","ingredients":["potato","tomato","olive oil","onion","garlic","paprika"],"prepTimeInMinutes":72}

Deleted 2 documents
```

Reply:
That output looks great to me, @ccho-mongodb!  Let's go with itT

Reply:
LGTM